api/responses/get-case-risk.json

```diff
@@ -15,17 +15,25 @@
     "riskMitigationFactors": "Factors include:\n- engagement in interventions related to mood/behaviour control\n- avoidance of drugs and alcohol\n- steady employment\n- stable accommodation\n- sources of support and value that are separate from being part of a gang/maintaining reputation e.g. family engagement, job, education",
     "riskOfSeriousHarm": {
       "overallRisk": "VERY_HIGH",
-      "riskToChildren": "LOW",
-      "riskToPublic": "VERY_HIGH",
-      "riskToKnownAdult": "MEDIUM",
-      "riskToStaff": "HIGH",
-      "lastUpdated": "2021-10-09"
+      "riskInCommunity": {
+        "riskToChildren": "MEDIUM",
+        "riskToPublic": "HIGH",
+        "riskToKnownAdult": "HIGH",
+        "riskToStaff": "VERY_HIGH"
+      },
+      "riskInCustody": {
+        "riskToChildren": "LOW",
+        "riskToPublic": "VERY_HIGH",
+        "riskToKnownAdult": "MEDIUM",
+        "riskToStaff": "HIGH",
+        "riskToPrisoners": "MEDIUM"
+      }
     }
```

Comment:
Might want to add a lastUpdatedDate to this part of the payload?

Reply:
it's there already, under `roshSummary` block